jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -302,7 +302,7 @@ void java_bytecode_convert_classt::convert(
     }
     catch(const unsupported_java_class_signature_exceptiont &e)
     {
-      warning() << "Class: " << c.name
+      debug() << "Class: " << c.name
```

Comment:
What's the verbosity level for debug?

Reply:
it's like this
1 - error
2 - warning
4 - result
6 - status
8 - stats
10 - debug